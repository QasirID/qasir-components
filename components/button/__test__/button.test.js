import React, { Component } from 'react';
import { render, shallow, mount } from 'enzyme';
import renderer from 'react-test-renderer';

import Button from '../button';

let wrapper;
beforeEach(() => {
    wrapper = render(<Button>Follow</Button>);
});
describe('Button', () => {
    it('Renders button type correctly', () => {
        const tree = renderer
            .create(<a page="http://www.facebook.com">Facebook</a>)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});