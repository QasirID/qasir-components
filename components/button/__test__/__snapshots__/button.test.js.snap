// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Renders button type correctly 1`] = `
<a
  page="http://www.facebook.com"
>
  Facebook
</a>
`;
